app/src/components/app.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import css from '../../styles/index.scss';
 
-const ACCESS_TOKEN_REGEX = /#access_token=([a-zA-Z0-9.\-\_]*)(&[a-z=])?/g;
+const ACCESS_TOKEN_REGEX = /#access_token=([a-zA-Z0-9.\-_]*)(&[a-z=])?/g;
```

Comment:
This was picked up by eslint, it's not necessary to escape `_` in regexes. If you feel this is unsafe I can disable the eslint rule for this line.